src/Either.js

```diff
@@ -127,4 +127,12 @@ Either.Left = function(value) {
 };
 
 
+// Fold
+Either.prototype.fold = function eitherFold(leftFn, rightFn) {
+  if (this instanceof _Left) {
+    return leftFn(this.value);
+  }
+  return rightFn(this.value);
+};
```

Comment:
Why not re-use the static `Either.either` function and further abstract the check for Left or Right?

```
Either.prototype.fold = function(leftFn, rightFn) {
  return Either.either(leftFn, rightFn)(this)
}
```